integration-tests/tests/bytecode_test.rs

```diff
@@ -0,0 +1,66 @@
+mod helpers;
+
+#[cfg(test)]
+pub mod test {
+    use std::{fs::File, io::BufReader};
+
+    use dotenv::dotenv;
+    use starknet::{providers::{JsonRpcClient, jsonrpc::HttpTransport}, core::{ types::{FieldElement}}};
+    use crate::helpers::{config::StarknetConfig, client::KakarotClient, utils::{hex_to_field_elements, TestCase}};
+
+
+    #[tokio::test]
+    async fn test_block_information() {
+    dotenv().ok();
+
+    let starknet_config = StarknetConfig::from_env().unwrap();
+    let starknet_provider = JsonRpcClient::new(HttpTransport::new(starknet_config.starknet_rpc));
+
+    let kakarot_client = KakarotClient::new(
+        starknet_provider,
+        starknet_config.kakarot_address,
+        starknet_config.proxy_account_class_hash
+    );
+
+    let file = match File::open("test_cases.json") {
```

Comment:
Maybe we would like to have the file directory as an .env variable, instead of hard coding?

Reply:
I think this file should be organized in a `data` directory and not be passed via the environment, as we might have other json files in the future.

I am putting them in a `data` directory inside `tests`, but let me know if you have other thoughts.